packages/suite/src/actions/wallet/signVerifyActions.ts

```diff
@@ -1,133 +1,130 @@
 import TrezorConnect from 'trezor-connect';
-import { validateAddress } from '@wallet-utils/ethUtils';
 import * as notificationActions from '@suite-actions/notificationActions';
-import { SIGN_VERIFY } from './constants';
-import { Dispatch, GetState } from '@suite-types';
+import { GetState, Dispatch } from '@suite-types';
+import { copyToClipboard } from '@suite-utils/dom';
 
-export type inputNameType =
-    | 'signAddress'
-    | 'signMessage'
-    | 'signSignature'
-    | 'verifyAddress'
-    | 'verifyMessage'
-    | 'verifySignature';
+export const copyAddress = (address: string) => (dispatch: Dispatch) => {
+    const errorResponse = copyToClipboard(address, null);
 
-export type SignVerifyActions =
-    | { type: typeof SIGN_VERIFY.SIGN_SUCCESS; signSignature: string }
-    | { type: typeof SIGN_VERIFY.CLEAR_SIGN }
-    | { type: typeof SIGN_VERIFY.CLEAR_VERIFY }
-    | { type: typeof SIGN_VERIFY.INPUT_CHANGE; inputName: inputNameType; value: string }
-    | { type: typeof SIGN_VERIFY.TOUCH; inputName: inputNameType }
-    | { type: typeof SIGN_VERIFY.ERROR; inputName: inputNameType; message?: string };
-
-export const sign = (path: [number], message: string, hex = false) => async (
-    dispatch: Dispatch,
-    getState: GetState,
-) => {
-    const selectedDevice = getState().suite.device;
-    if (!selectedDevice) return;
-
-    const response = await TrezorConnect.ethereumSignMessage({
-        device: {
-            path: selectedDevice.path,
-            instance: selectedDevice.instance,
-            state: selectedDevice.state,
-        },
-        path,
-        hex,
-        message,
-        useEmptyPassphrase: selectedDevice.useEmptyPassphrase,
-    });
-
-    if (response && response.success) {
-        dispatch({
-            type: SIGN_VERIFY.SIGN_SUCCESS,
-            signSignature: response.payload.signature,
-        });
-    } else {
+    if (typeof errorResponse === 'string') {
         dispatch(
```

Comment:
you dont have to handle this error, just do nothing (like other implementations of `copyToClipboard`

Reply:
same as above